src/VirtualMonoRepo/source-mappings.json

```diff
@@ -108,7 +108,10 @@
         {
             "name": "nuget-client",
             "defaultRemote": "https://github.com/NuGet/NuGet.Client",
-            "defaultRef": "dev"
+            "defaultRef": "dev",
+            "exclude":[
+                "src/NuGet.Clients/NuGet.VisualStudio.Client/*"
```

Comment:
What would happen if you put `src/NuGet.Clients/NuGet.VisualStudio.Client`?

Reply:
works too

Reply:
Let's do that then